cgi/manifest.pl

```diff
@@ -43,37 +43,40 @@
 ProductOpener::Display::init();
 
 my $short_name = lang("site_name");
-my $long_name = $short_name;
+my $long_name  = $short_name;
 
 # https://stackoverflow.com/a/16533563/11963
-$short_name =~ s/\b([A-Z])[a-z]+(?=\s+[A-Z][a-z])|\G(?!^)\s+([A-Z])[a-z]+/$1$2/g;
+$short_name =~
+  s/\b([A-Z])[a-z]+(?=\s+[A-Z][a-z])|\G(?!^)\s+([A-Z])[a-z]+/$1$2/g;
 
-if ($cc eq 'world') {
-	$long_name .= " " . uc($lc);
-	$short_name .= " " . uc($lc);
+if ( $cc eq 'world' ) {
+    $long_name  .= " " . uc($lc);
+    $short_name .= " " . uc($lc);
 }
 else {
-	$long_name .= " " . uc($cc) . "/" . uc($lc);
-	$short_name .= " " . uc($cc) . "/" . uc($lc);
+    $long_name  .= " " . uc($cc) . "/" . uc($lc);
+    $short_name .= " " . uc($cc) . "/" . uc($lc);
 }
 
 my %manifest = (
-	lang => $lc,
-	name => $long_name,
-	short_name => $short_name,
-	description => lang('site_description'),
-	start_url => format_subdomain($subdomain),
-	scope => '/',
-	display => 'standalone',
-	prefer_related_applications => $JSON::PP::true,
+    lang                        => $lc,
```

Comment:
The extra spacing before the => to align them looks different from some other PRs, like this one:

https://github.com/openfoodfacts/openfoodfacts-server/pull/7056/files

Where we have:

```
	display_page(
		{
			title => lang('session_title'),
			content_ref => \$html
		}
	);
```